lib/schemas/CssColor.json

```diff
@@ -0,0 +1,221 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
```

Comment:
```suggestion
    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Reply:
I could change that, but I also need to change `definitions` to `$def`